samples/Perl 6/chromosome.pl

```diff
@@ -0,0 +1,9 @@
+class Chromosome {
+  has Seq $.chromosome is rw;
+  has $.fitness is rw;
+
+}
+
+my $len = 32;
+my $this-chromosome = Chromosome.new( chromosome => map( { rand >= 0.5 ?? True !! False }, 1..$len )  );
+say $this-chromosome.chromosome();
```

Comment:
Is this file somehow uniquely Perl 6 such that it can clearly be differentiated from the exact same code written in Perl 5, or should this be added to the `samples/Perl/`?

The reason I ask is we use these samples to train the classifier and if this is identical to Perl 5, it's likely to add noise to the Perl 6 side of the classifier when it comes down to a Perl 5 vs Perl 6 comparison.

Reply:
It could also go into `tests/fixtures/Perl 6/` if we don't want it as a sample.

Reply:
On a superficial level, it could look similar to Perl. It's been a long time, but if I remember correctly I included it here precisely for that reason. It could go to fixtures alright. Just let me know what to do, and I'll do it straight away.

Reply:
Ok. If this is just to keep the tests passing, then yup, `tests/fixtures/Perl 6/` is definitely the best place for it.

Reply:
Done. I'm getting some local errors which apparently have nothing to do with moving stuff around. Let me know if there's something I can do about that.